Jellyfin.Plugin.SmartLists/Configuration/config-filters.js

```
@@ -165,11 +172,23 @@
             }
 
-            // Search in username (resolved from User ID)
-            if (page && page._usernameCache && playlist.UserId) {
-                const username = page._usernameCache.get(playlist.UserId);
-                if (username && username.toLowerCase().indexOf(searchTerm) !== -1) {
-                    return true;
+            // Search in username (resolved from User ID or UserPlaylists)
+            if (page && page._usernameCache) {
+                // Check UserPlaylists array (multi-user playlists)
+                if (playlist.UserPlaylists && playlist.UserPlaylists.length > 0) {
+                    for (var i = 0; i < playlist.UserPlaylists.length; i++) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)